src/components/layout/index.tsx

```diff
@@ -1,18 +1,24 @@
+import type { FooterProps } from './footer';
 import { Footer } from './footer';
 import { SimpleFooter } from './footer/simple-footer';
 import { Header } from './header';
 
-interface LayoutProps {
+interface LayoutProps extends FooterProps {
```

Comment:
Let's pick just the prop needed, this way, if other props are added to the footer later, they won't be available here. 

```suggestion
interface LayoutProps extends Pick<FooterProps, "withoutNewsletter"> {
```